Robot-Tests/testcases_Paallyste.robot

```diff
@@ -0,0 +1,40 @@
+# Joni Laari        Sitowise Oy     2022
+
+*** Variables ***
+
+*** Settings ***
+Documentation       Regression testcases for Digiroad
+Resource            common_keywords.robot
+
+Suite Setup         Login To DigiRoad
+Suite Teardown      Close Browser
+
+Test Setup          Testin Aloitus
+
+
+*** Test Cases ***
+Päällyste 1
+    [Tags]  AWS  Päällyste
+    [Documentation]  Päällysteen tarkastelu katselumoodissa.
+
+    KW_Paallyste.Päällyste_1      6888907, 429796
+
+
+Päällyste 2
```

Comment:
Mun mielestä nää vois nimetä kuvaavasti, esim. Päällysteen muokkaus, eikä näitä "tietolaji 1,2,3...".

Reply:
Samaa mieltä, seurasin tässä aiempien testien käytäntöä